### ISSUE TITLE
Is there a problem with "space_after_anon_function"?

### ISSUE THREAD

**Issue by NerdPad** [Glavin001/atom-beautify#824]
### Description

The "space_after_anon_function" option doesn't seem to work.

I have "space_after_anon_function" set to false in my ~/.jsbeautifyrc, and "JavaScript - space after anon function" unchecked.

### Expected Results

The beautified code should have looked like:

```
function() {
};
```

### Actual Results

The beautified code looks like:

```
function () {
};
```

### Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

### Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/NerdPad/f1526fa3426123ec5f8a

**Comment by Glavin001** [comment]
Please follow this template, specifically adding back the missing `Debug` section: https://github.com/Glavin001/atom-beautify/blob/master/ISSUE_TEMPLATE.md#debug
Thank you.

**Comment by NerdPad** [comment]
@Glavin001 Thank you for replying. I have added debug section in the original post.

**Comment by Glavin001** [comment]
tl;dr: Change `jslint_happy` to `false`.

---

I can confirm that `space_after_anon_function` is `false` from your debugging logs: https://gist.github.com/NerdPad/f1526fa3426123ec5f8a#final-options

Looking at js-beautify's internal code: https://gist.github.com/NerdPad/f1526fa3426123ec5f8a#final-options

```
    space_after_anon_function (default false) - should the space before an anonymous function's parens be added, "function()" vs "function ()",
          NOTE: This option is overriden by jslint_happy (i.e. if jslint_happy is true, space_after_anon_function is true by design)
```

And `jslint_happy` is set to `true`. Change it to `false` and then your `space_after_anon_function` should not be ignored.

**Comment by NerdPad** [comment]
Thank you. That's what I was missing.
